src/processors/BaseProcessor.cpp

```diff
@@ -241,3 +241,36 @@ juce::Point<int> BaseProcessor::getPosition (Rectangle<int> parentBounds)
 {
     return (editorPosition * juce::Point { (float) parentBounds.getWidth(), (float) parentBounds.getHeight() }).toInt();
 }
+
+void BaseProcessor::makeBufferMono(AudioBuffer<float>& buffer)
```

Comment:
So this could be a static method, but unfortunately I'm not so sure its reuse would be that helpful outside of the stereo merger block.

Reply:
Yeah definitely, I moved it back to stereo merger